src/generic_core/core.ts

```diff
@@ -55,6 +55,9 @@ class UIConnector implements uProxy.UIAPI {
             networkNames: Object.keys(Social.networks),
             globalSettings: core.globalSettings
           });
+      for (var networkName in Social.networks) {
+        Social.notifyUI(networkName);
+      }
```

Comment:
Or a better question: if we need to send all the social network info at the same time as sending the UI state, can we not just combine it into 1 update (INITIAL_STATE), rather than INITIAL_STATE, plus another message for each network, plus another message for each user on the logged in networks?